contrib/src/main/java/com/datatorrent/contrib/cassandra/AbstractCassandraInputOperator.java

```diff
@@ -112,8 +115,15 @@ public void emitTuples()
     String query = queryToRetrieveData();
     logger.debug("select statement: {}", query);
 
+    SimpleStatement stmt = new SimpleStatement(query);
+    stmt.setFetchSize(fetchSize);
```

Comment:
The query is fetched each time "queryToRetrieveData()". In current implementation query doesn't change much. But anyone can implement that function to change query parameters in each run. In that case we need to create statement using the current query. This is abstract class so trying to make sure it helps in all known use cases.